parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
RE side effects: I'll move the replacement step outside of the function call in a side-effect loop, i.e.
```py
    object_tags = html.find_all("object", data=True)
    objs_as_imgs = [convert_obj_to_img_tag(html, o) for o in object_tags]
    for obj, img in zip(object_tags, objs_as_imgs):
        obj.replaceWith(img)
```